.gitignore

```diff
@@ -118,3 +118,4 @@ env/
 
 ### Configuration files ###
 .env
+/.env
```

Comment:
Good point. This is because I use a gitignore extension for VSCode and I automatically added the .env folder to the gitignore. It didn't check if it was already in there. Should have checked.